### PR TITLE
VSO time attribute to return Astropy time object

### PR DESCRIPTION
**Pull request by nitinkgp23**
This PR is in progress. Aim is to modify all those codes where the tests fail. Hence, this PR.
Please don't review until all tests pass.

### CODE REVIEW

**Comment by tannmay** [comment]
I am taking this forward

File: sunpy/net/vso/attrs.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 from datetime import datetime
 
 from astropy import units as u
+from astropy.time import Time as astropyTime
```
Comment (dpshelio) [inline sunpy/net/vso/attrs.py]: why not just import it as `Time`?

Comment (tannmay) [reply sunpy/net/vso/attrs.py]: Because it collides with sunpy's time.
Same name.

Comment (dpshelio) [reply sunpy/net/vso/attrs.py]: Ooohh, I see, vso's `Time`.

File: sunpy/net/vso/attrs.py [hunk 2]
```diff
@@ -178,12 +179,15 @@ def __init__(self, start, end=None, near=None):
             self.start = start.start
             self.end = start.end
         else:
-            self.start = parse_time(start)
-            self.end = parse_time(end)
+            self.start = start if isinstance(start, astropyTime) else astropyTime(parse_time(start))
+            self.end = end if isinstance(end, astropyTime) else astropyTime(parse_time(end))
```
Comment (dpshelio) [inline sunpy/net/vso/attrs.py]: would this be nicer?
```python
self.start = isinstance(start, astropyTime) and start or astropytime(parse_time(start))
```

**Comment by nabobalis** [comment]
Would this not end up as part of the GSOC astropy time projet looking forward? 

Safe to close?

**Comment by Cadair** [comment]
Yeah this can probably be closed. Might be worth adding a link to it from the relevant issues / project description.

**Comment by nabobalis** [comment]
Sent PR to website.

**Pull request closed without merging**